### PR TITLE
created catalog

### PR DESCRIPTION
**Pull request by ihorandrianov**
- [DEMO LINK](https://ihorandrianov.github.io/layout_catalog/)
- [TEST REPORT LINK](https://ihorandrianov.github.io/layout_catalog/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -4,9 +4,518 @@
...
+            <p class="card__price-price">
+              Price:
+            </p>
+            <p class="card__price-numbers">
+              $2,199
+            </p>
+          </div>
+
+
+          <a
```
Comment (Anna-Borodaienko) [inline src/index.html:147]:
```suggestion
          </div>

          <a
```

File: src/index.html [hunk 2]
```diff
@@ -4,9 +4,518 @@
...
+            href="/"
+            class="card__btn"
+            data-qa="card-hover"
+          >
+            Buy
+          </a>
+        </div>
+
+
+        <div class="card">
```
Comment (Anna-Borodaienko) [inline src/index.html:157]:
```suggestion
        </div>

        <div class="card">
```

File: src/styles/blocks/card/_card.scss [hunk 3]
```diff
@@ -0,0 +1,125 @@
+@import "../../utils/variables";
+@import "../../utils/mixins";
+
+.product-list {
+  display: flex;
+  justify-content: center;
+}
+
+.product-list__container {
+  display: flex;
```
Comment (Anna-Borodaienko) [inline src/styles/blocks/card/_card.scss:10]:
```suggestion
.product-list {
  display: flex;
  justify-content: center;


  &__container {
    display: flex;
  
}
```

File: src/styles/blocks/card/_card.scss [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+  }
+
+  &__body {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+    width: 100%;
+
+    padding: 0 16px 24px;
```
Comment (Anna-Borodaienko) [inline src/styles/blocks/card/_card.scss:58]:
```suggestion
    width: 100%;
    padding: 0 16px 24px;
```

File: src/styles/utils/_mixins.scss [hunk 5]
```diff
@@ -0,0 +1,12 @@
+@mixin default-text {
+  font-family: "Roboto", Helvetica, sans-serif;
```
Comment (Anna-Borodaienko) [inline src/styles/utils/_mixins.scss:2]: We have this font-family on all our page. I'd add this line not to mixin, but to tag html

File: src/styles/blocks/card/_card.scss [hunk 6]
```diff
@@ -0,0 +1,125 @@
...
+    transition: 0.3s;
+  }
+
+  &__btn:hover {
+    background-color: $btn-secondary;
+    color: $secondary-color;
+    border: 1px solid $secondary-color;
+  }
+
+}
```
Comment (Anna-Borodaienko) [inline src/styles/blocks/card/_card.scss:116]:
```suggestion
  }
}
```

File: src/styles/blocks/card/_card.scss [hunk 7]
```diff
@@ -0,0 +1,125 @@
...
+
+.card:nth-last-child(-n + 4) {
+  margin-top: 46px;
+  order: 2;
+}
```
Comment (Anna-Borodaienko) [inline src/styles/blocks/card/_card.scss:121]: Why do you add margin-top to last 4 elements? You have to add them margin-bottom 0px, as I understand the task.

**Review by Anna-Borodaienko — COMMENTED** [review]
Deleted so many folders for blocks, please. You need 1 folder - Blocks with all these files

**Review by MaksimusCRS — CHANGES_REQUESTED** [review]
I removed 2 cards and something went wrong 🥲 
image

**Comment by Anna-Borodaienko** [comment]
Of course! Because you add an additional margin-top to last 4 elements. Try to set all margins and paddings to the whole card. Your card's  elements must have only margin-bottom.

**Review by MaksimusCRS — CHANGES_REQUESTED** [review]
Still doesn't working 
image

**Comment by ihorandrianov** [comment]
> Still doesn't working
> image

forget to deploy. Now should work

File: src/index.html [hunk 8]
```diff
@@ -4,9 +4,509 @@
...
+            </a>
+          </li>
+          <li class="nav__li">
+            <a
+              class="nav__link"
+              href="/"
+            >
+              Smartphones
+            </a>
+          </li>
```
Comment (MaksimusCRS) [inline src/index.html:56]: Checklist #2

File: src/styles/blocks/card/_card.scss [hunk 9]
```diff
@@ -0,0 +1,136 @@
...
+    line-height: 16px;
+
+    transition: 0.3s;
+  }
+
+  &__btn:hover {
+    background-color: $btn-secondary;
+    color: $secondary-color;
+    border: 1px solid $secondary-color;
+  }
```
Comment (MaksimusCRS) [inline src/styles/blocks/card/_card.scss:135]: Checklist #10

**Review by MaksimusCRS — CHANGES_REQUESTED** [review]

File: src/styles/utils/_variables.scss [hunk 10]
```diff
@@ -5,5 +5,5 @@ $secondary-color: #00acdc;
 $btn-secondary: #fff;
 
 $card-width: 200px;
-$card-margin: $card-width * 0.25 - 2;
-$section-width: 1000px;
+$card-margin: 50px;
+$section-width: $card-width * 4 + $card-margin * 3;
```
Comment (MaksimusCRS) [inline src/styles/utils/_variables.scss:9]: Add variables for 4 and 3, i don't know why you use them

**Review by MaksimusCRS — APPROVED** [review]

**Pull request closed without merging**